plugins/woocommerce/client/admin/client/marketing/components/PluginCardBody/useIsPluginInstalledNotActivated.ts

```diff
@@ -1,24 +0,0 @@
-/**
- * External dependencies
- */
-import { useCallback } from '@wordpress/element';
-import { useSelect } from '@wordpress/data';
-import { PLUGINS_STORE_NAME } from '@woocommerce/data';
-
-export const useIsPluginInstalledNotActivated = () => {
-	const { getPluginInstallState } = useSelect( ( select ) => {
-		return {
-			getPluginInstallState:
-				select( PLUGINS_STORE_NAME ).getPluginInstallState,
-		};
-	} );
```

Comment:
The code here was returning the `getPluginInstallState` selector function but not calling the function. This is not a good practice and can be buggy. See [Best practices for using useSelect() from @wordpress/data
](https://jardasn.blog/2021/01/22/some-best-practices-for-using-useselect-from-wordpress-data/) (look for "Always call selector functions inside the callback").

That's why I have refactored, simplified and removed this `useIsPluginInstalledNotActivated` hook in [`36e0012` (#54485)](https://github.com/woocommerce/woocommerce/pull/54485/commits/36e0012e8df35d5d1697c1a846c1b8999a12c972).